lm_eval/models/vllm_causallms.py

```diff
@@ -209,7 +210,10 @@ def __init__(
             add_bos_token=add_bos_token,
         )
         self.tokenizer = configure_pad_token(self.tokenizer, model_config=self._config)
-        self.enable_thinking = enable_thinking
+        self.chat_template_args = chat_template_args or {}
+        self.enable_thinking = chat_template_args.pop(
```

Comment:
@Avelina9X This should be `self.chat_template_args.pop(` instead of `chat_template_args.pop(`, right?

Reply:
will be fixed by https://github.com/EleutherAI/lm-evaluation-harness/pull/3182